test/specs/apexActions.e2e.ts

```diff
@@ -0,0 +1,211 @@
+      // Go to apex class file
+      const workbench = utilities.getWorkbench();
+      const editor = await utilities.getTextEditor(workbench, 'PropertyController.cls');
+      await editor.moveCursor(10, 30);
+      const contextMenu = await editor.openContextMenu();
+      await contextMenu.select('SFDX: Create Apex Action from This Class');
+
+      // Verify 'SFDX: Create Apex Action from This Class' generated the open api file
+      const editorView = workbench.getEditorView();
+      const activeTab = await editorView.getActiveTab();
+      const title = await activeTab?.getTitle();
+      expect(title).to.equal('PropertyController_openapi.yml');
+
+      // Verify file content
+      const textEditor = await utilities.getTextEditor(workbench, title!);
+      const expectedTexts = [
+        'openapi: 3.0.0',
+        'info:\n\ttitle:',
+        '\tversion',
+        'paths:\n\t',
+        '/getPagedPropertyList:\n\t\tpost:',
+        '\t\t\tsummary:',
+        '\t\t\tparameters:',
+        '\t\t\t\t- name: searchKey',
+        '\t\t\t\t\tin: query',
+        '\t\t\t\t\trequired: true',
+        '\t\t\t\t\tdescription:',
+        '\t\t\t\t\tschema:\n\t\t\t\t\t\ttype: string',
+        '\t\t\t\t- name: maxPrice',
+        '\t\t\t\t- name: minBedrooms',
+        '\t\t\t\t- name: minBathrooms',
+        '\t\t\t\t- name: pageSize',
+        '\t\t\t\t- name: pageNumber',
+        '/getPictures:\n\t\tpost:',
+        '\t\t\t\t- name: propertyId',
+        '\t\t\tresponses'
+      ];
+      verifyFileContent(await textEditor.getText(), expectedTexts);
+    });
+  }
+
+  if (process.platform !== 'darwin') {
+    step('SFDX: Create Apex Action from This Class - Explorer', async () => {
+      utilities.log(`${testSetup.testSuiteSuffixName} - SFDX: Create Apex Action from This Class - Explorer`);
+      // Close current editor
+      await utilities.closeCurrentEditor();
+
+      // Go to apex class file
+      const workbench = await utilities.getWorkbench();
+      const sidebar = await workbench.getSideBar().wait();
+      const content = await sidebar.getContent().wait();
+      const treeViewSection = await content.getSection(testSetup.tempProjectName);
+      if (!treeViewSection) {
+        throw new Error(
+          'In verifyProjectLoaded(), getSection() returned a treeViewSection with a value of null (or undefined)'
+        );
+      }
+
+      const objectTreeItem = (await treeViewSection.findItem('PropertyController.cls')) as DefaultTreeItem;
+      if (!objectTreeItem) {
+        throw new Error(
+          'In verifyProjectLoaded(), findItem() returned a forceAppTreeItem with a value of null (or undefined)'
+        );
+      }
+
+      expect(objectTreeItem).to.not.be.undefined;
+      await (await objectTreeItem.wait()).expand();
+
+      const contextMenu = await objectTreeItem.openContextMenu();
+      await contextMenu.select('SFDX: Create Apex Action from This Class');
```

Comment:
Is this the context menu that we can see when right clicking in the file explorer?